cassandra/io/eventletreactor.py

```diff
@@ -0,0 +1,213 @@
+#
+# Copyright 2014 Nebula, Inc
+# Copyright 2014 Justin Santa Barbara
+# Copyright 2014 Symantec Corporation
+# Copyright 2013-2014 DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import collections
+import errno
+import functools
+import logging
+import os
+import threading
+
+import eventlet
+from eventlet import green
+from eventlet import queue
+
+
+try:
+    from cStringIO import StringIO  # ignore flake8 warning: # NOQA
+except ImportError:
+    from StringIO import StringIO  # ignore flake8 warning: # NOQA
```

Comment:
Good note. I was actually trying that myself when the second point occurred to me.